records_mover/session.py

```diff
@@ -194,6 +217,44 @@ def _boto3_session(self) -> Optional['boto3.session.Session']:
         else:
             return self.creds.boto3_session(self._default_aws_creds_name)
 
+    def _gcs_creds(self) -> Optional['google.auth.credentials.Credentials']:
+        try:
+            import google.auth.exceptions
+            if self._default_gcp_creds_name is None:
+                import google.auth
+                credentials, project = google.auth.default()
+                return credentials
+            else:
+                return self.creds.gcs(self._default_gcp_creds_name)
+        except OSError:  # TODO: google.auth.exceptions.DefaultCredentialsError):
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker